bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/DiscoveryResultImpl.java

```diff
@@ -226,4 +239,14 @@ public long getTimestamp() {
     public long getTimeToLive() {
         return timeToLive;
     }
+
+    @Override
+    public void discoveredBy(String discoveryService) {
+        this.discoveredBy = discoveryService;
+    }
+
+    @Override
+    public String isDiscoveredBy() {
```

Comment:
I think the proper English would be "hasBeenDiscoveredBy". But to make it explicit which of these two methods does what: Wouldn't it be simpler to treat it like a normal bean property and call the methods `setDiscoverer` and `getDiscoverer`?